### PR TITLE
Void decisions

### PR DESCRIPTION
**Pull request by rperez89**
This closes <https://github.com/1Hive/gardens/issues/511>

Screen Shot 2022-02-02 at 5 23 20 PM

Screen Shot 2022-02-02 at 5 23 06 PM

### CODE REVIEW

File: src/hooks/useProposals.js [hunk 1]
```diff
@@ -124,9 +125,12 @@ export function useProposal(proposalId, appAddress) {
     proposalId,
     appAddress
   )
+  const connectedGarden = useConnectedGarden()
+  console.log('CONNECTED GARDEN ', connectedGarden)
```
Comment (kamikazebr) [inline src/hooks/useProposals.js:129]: Maybe remove the log here?

File: src/hooks/useProposals.js [hunk 2]
```diff
@@ -287,7 +293,8 @@ function processDecision(proposal) {
     endDate,
     proposal.challengeEndDate
   )
-  const statusData = getVoteStatusData(proposal, hasEnded, PCT_BASE)
+  const statusData = getVoteStatusData(proposal, hasEnded, PCT_BASE, voidedDecisions)
+  console.log('PROPOSAL WITH DATa ', statusData)
```
Comment (kamikazebr) [inline src/hooks/useProposals.js:297]: Here too

File: src/components/Garden/DecisionDetail/VoteStatus.tsx [hunk 3]
```diff
@@ -114,6 +114,15 @@ export const getStatusAttributes = (
       background: theme.background,
     }
   }
+  if (statusData.voided) {
+    return {
+      label: 'Voided',
```
Comment (kamikazebr) [inline src/components/Garden/DecisionDetail/VoteStatus.tsx:119]: we could use the constant PROPOSAL_STATUS_VOIDED_STRING here?

File: src/hooks/useProposals.js [hunk 4]
```diff
@@ -277,7 +283,7 @@ function processProposal(proposal, latestBlock, account, config) {
   }
 }
 
-function processDecision(proposal) {
+function processDecision(proposal,voidedDecisions) {
```
Comment (kamikazebr) [inline src/hooks/useProposals.js:286]: Your Prettier its off? I think with save it should get automatic space afte comma.

**Comment by kamikazebr** [comment]
Weird cant access the tec gardens on staging.

on the master below

image

On the branch
image

someone know why?

I'll try localhost

File: src/hooks/useProposals.js (hunk 1 above)
Comment (rperez89) [reply src/hooks/useProposals.js:129]: oops

**Comment by kamikazebr** [comment]
I found you modified the dao-list repo to save the voided and there i found the garden to test 

http://localhost:3000/#/rinkeby/garden/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a

looks great! That was fast feature implementation. Good job! @rperez89

File: src/hooks/useProposals.js (hunk 4 above)
Comment (rperez89) [reply src/hooks/useProposals.js:283]: > I found you modified the dao-list repo to save the voided and there i found the garden to test
> 
> http://localhost:3000/#/rinkeby/garden/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a
> 
> looks great! That was fast feature implementation. Good job! @rperez89

ohh sorry i should have explainded how to test it

**Review by kamikazebr — APPROVED** [review]
Awesome improvement!

**Review by tmoutinho — APPROVED** [review]
✌️

**Review by 0xGabi — APPROVED** [review]
Well done!

**Pull request merged**